package.json

```diff
@@ -137,9 +137,9 @@
     "why-did-you-update": "1.0.8"
   },
   "resolutions": {
-    "@types/relay-runtime": "1.3.10",
-    "eslint": "^6.8.0",
-    "chokidar": "^3.0.0"
+    "@types/relay-runtime": "1.3.0",
+    "eslint": "6.8.0",
+    "chokidar": "3.5.3"
```

Comment:
This update may be contributing to Jenkin's failure, can this be avoided? If not we can also look at remove the jenkinsfile since it just repeats what travis already does.

Reply:
@kulgan Yes, @philtom-ctds mentioned that removal of jenkins integration is in process so he's gonna check further and remove the check after verifying it once.

Reply:
@kulgan let's remove the jenkins integration for portal-ui  👍